### ISSUE TITLE
Instant Crusader crash when trying to use AIV features from the patched AIV Editor

### ISSUE THREAD

**Issue by Heroesflorian** [Sh0wdown/UnofficialCrusaderPatch#208]
I just downloaded the patched village.exe (from the 2.07 release, as I found no newer one), copied the entire AIV Editor folder, replaced the original with the patched village.exe there, then copied an existing AIV file with the patched version of the AIV Editor and placed a couple additional siege engine spots on that AIV (to have 9-10 spots for each of the four possible siege engine types), and saved the AIV file. 

Then I copied that modified AIV file from the AIV Editor's village directory to the Crusader aiv directory and launched Crusader (patched with 2.09 release), started a game with that AIV, and as soon as the engineers guild was built (which also is the point from that on siege engines / tents can be placed by the AI), Crusader crashed instantly. 
So I restarted Crusader, restarted a match with that same AIV, and it crashed again as soon as the engineers guild was built... I tried a couple more times, but it crashed each and every time. 

Did I make any mistake in the patched AIV Editor setup process? Is the 2.07 release village.exe only compatible with Crusader with patch 2.07? Or are those additional siege engine spots just very buggy?

**Comment by Monsterfisch** [comment]
probably a good idea to attach the save file

**Comment by Heroesflorian** [comment]
> probably a good idea to attach the save file

Indeed, although I am trying to narrow it down a bit first... on a differently-modified version of the same base AIV, it worked (or didn't crash yet, at least), so I am doing a couple versions with different modifications now... although I probably won't get to finish those tests until tomorrow evening.

**Comment by ByBurton** [comment]
I had that problem as well. Try only placing 9 fire ballista and trebuchets at max.

**Comment by Sh0wdown** [comment]
> Try only placing 9 fire ballista and trebuchets at max.

Yea, there seem to be some more limits than I found back then, so try to do this.

> Is the 2.07 release village.exe only compatible with Crusader with patch 2.07?

Should work with any version.

**Comment by Heroesflorian** [comment]
@ByBurton That did indeed seem to be the issue... 10th Tre / FBal spots. 

Still, the other slots are kinda buggy and unreliable so far... pity.

**Comment by ByBurton** [comment]
@Heroesflorian Could you please change the title of this issue so it reflects the problem with trebuchets, mangonels, fire ballistae and towre ballistae?
